frontend/store/surplus.ts

```diff
@@ -103,7 +122,9 @@ export const actions = {
         try {
             commit('setAuctionsFetching', true);
             const auctions = await fetchActiveSurplusAuctions(network);
-            auctions.forEach(auction => commit('addAuctionToStorage', auction));
+            const enrichedAuctionPromises = auctions.map(auc => enrichSurplusAuctionWithMarketPrice(network, auc));
+            const auctionTransactions = await Promise.all(enrichedAuctionPromises);
+            auctionTransactions.forEach(auctionTransaction => commit('addAuctionToStorage', auctionTransaction));
```

Comment:
As suggested by @zoey-kaiser, please enrich surplus auction in one step via a single `enrichSurplusAuction` function that does all kind of enrichments